src/screens/UnknownTokens.js

```diff
@@ -249,8 +249,10 @@ class UnknownTokens extends React.Component {
 
       return unknownTokens.map((token, index) => {
         const isNFT = helpers.isTokenNFT(token.uid, this.props.tokenMetadata);
-        const tokenBalance = get(this.props.tokensBalance, `${token.uid}`, { status: TOKEN_DOWNLOAD_STATUS.LOADING });
-        const tokenHistory = get(this.props.tokensHistory, `${token.uid}`, { status: TOKEN_DOWNLOAD_STATUS.LOADING });
+
+        // Note: We can't default to LOADING here otherwise we will never show the `Show History` button
```

Comment:
Why? This would be "LOADING" only until the balance/history if fetched.

Reply:
No, since we only load the history after the user clicks the "Show history" button and we hide that button when the history is already loading, the loadHistory action will never be triggered and the UI will show "Loading history..." forever